stop_radar.sh

```diff
@@ -7,5 +7,13 @@ do
 	atrm $i
 done  
 
-screen -X -S borealis quit
+NOW=`date +'%Y%m%d %H:%M:%S'`
 
+screen -X -S borealis quit | tee -a /data/borealis_logs/start_stop.log
+
+retVal=$?
```

Comment:
This is a great idea, only one thing that may not work as intended though with the $?: I think this will return the exit value of 'tee' (just whatever command was last I believe)

Try this to verify: 

```

kevin@aware:~> false | true
kevin@aware:~> echo $?
0
kevin@aware:~> true | false
kevin@aware:~> echo $?
1

```

Reply:
Makes sense. In my tests the screen quit didn't output anything anyway, so I think I'll just remove the tee.

Reply:
yep, one way around this is to capture the output of the 'screen -X -S borealis quit' by the following:

screen_quit_output=$((screen -X -S borealis quit))
retval=$?
echo $screen_quit_output >> /data/borealis_logs/start_stop.log